src/toil/statsAndLogging.py

```diff
@@ -112,9 +112,10 @@ def callback(fileHandle):
                 pass
             else:
                 def logWithFormatting(jobStoreID, jobLogs):
-                    logFormat = '\n%s    ' % jobStoreID
                     logger.debug('Received Toil worker log. Disable debug level '
-                                 'logging to hide this output\n%s', logFormat.join(jobLogs))
+                                 'logging to hide this output')
+                    for line in jobLogs:
+                        logger.debug('%s    %s', jobStoreID, line)
```

Comment:
Doh, I didn't notice that location. Yes, feel free to merge and modify later, or push to this PR, or anything really.